<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialogWindow</class>
 <widget class="QDialog" name="InfoDialogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 255, 255);</string>
  </property>
  <widget class="QDialogButtonBox" name="OkButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
font: 16pt &quot;Franklin Gothic Heavy&quot;;
width:140px;

border-radius: 25px;
background-color: rgb(85, 170, 255);
color: rgb(85, 85, 255);

}

QPushButton:hover{
background-color: rgb(50, 110, 255);
color: rgb(255, 255, 255);

}

QPushButton:pressed{
background-color: rgb(100, 170, 255);
}

</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="simple_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
image: &quot;C:\\Users\\Alex\\Downloads\\pngegg.png&quot;;
font: 22pt &quot;Franklin Gothic Heavy&quot;;

border-radius: 25px;
	background-color: rgb(255, 0, 0);
color: rgb(85, 85, 255);

}

QPushButton:hover{
background-color: rgb(50, 110, 255);
color: rgb(255, 255, 255);

}

QPushButton:pressed{
background-color: rgb(100, 170, 255);
}

</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="modern_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
image: &quot;C:\\Users\\Alex\\Downloads\\pngegg.png&quot;;
font: 22pt &quot;Franklin Gothic Heavy&quot;;

border-radius: 25px;
	background-color: rgb(255, 0, 0);
color: rgb(85, 85, 255);

}

QPushButton:hover{
background-color: rgb(50, 110, 255);
color: rgb(255, 255, 255);

}

QPushButton:pressed{
background-color: rgb(100, 170, 255);
}

</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="azer_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
image: &quot;C:\\Users\\Alex\\Downloads\\pngegg.png&quot;;
font: 22pt &quot;Franklin Gothic Heavy&quot;;

border-radius: 25px;
	background-color: rgb(255, 0, 0);
color: rgb(85, 85, 255);

}

QPushButton:hover{
background-color: rgb(50, 110, 255);
color: rgb(255, 255, 255);

}

QPushButton:pressed{
background-color: rgb(100, 170, 255);
}

</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(184, 248, 255);
font: 14pt &quot;Franklin Gothic Heavy&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OkButton</sender>
   <signal>accepted()</signal>
   <receiver>InfoDialogWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OkButton</sender>
   <signal>rejected()</signal>
   <receiver>InfoDialogWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
